docarray/base_doc/doc.py

```diff
@@ -28,7 +29,7 @@ class BaseDoc(BaseModel, IOMixin, UpdateMixin, BaseNode):
     class Config:
         json_loads = orjson.loads
         json_dumps = orjson_dumps_and_decode
-        json_encoders = {dict: orjson_dumps}
+        json_encoders = {AbstractTensor: lambda x: x}
```

Comment:
can u add a comment explaning why the `lambda x: x` is needed here ?

Smth like: 

Our custom orjson response is able to handle tensor by itself, therefore we don't need FastAPI to do any transformation to the tensor object but just gave it to the response class